### DISCUSSION TITLE
How to subscribe Appsync endpoint from rust client using aws-sdk-appsync or aws-sdk-amplify

### DISCUSSION THREAD

**Discussion by pankajb** [awslabs/aws-sdk-rust#801] [Q&A] [answered]
I am tring to subscribe to the endpoint hosted in AppSync and trying to mimic what is given here 
https://docs.amplify.aws/lib/graphqlapi/subscribe-data/q/platform/js/

How can I subscribe to the events getting pushed from AppSync endpoint?

**Comment by jdisanti** [comment, marked as answer]
The Amplify API hasn't been ported to Rust, so I don't think there's a way to do this currently.

**Reply by pankajb** [reply]
Thank you, if possible can you suggest please any graphQL client for rust which can potentially work with AppSync subscription? 
Have you come across anything? :)

**Reply by jdisanti** [reply]
Sorry, I don't have any insight here.
